src/style.css

```
@@ -0,0 +1,89 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Book.ttf");
+  font-weight: normal;
```

```suggestion
  font-weight: 300;
```
it can help to fix tests